release_notes/374-release-notes.adoc

```diff
@@ -348,15 +349,27 @@ If you are using the in-product docs, you can instead download the link:https://
 [id="resolved-in-version-3740"]
 === Resolved in version 3.74.0
 
+Release date: 27 February 2023
+
 * The {product-title-short} Operator previously incorrectly removed owner references from the `stackrox-db` persistent volume claim (PVC). This can cause an error such as `Failed reconciling PVC "stackrox-db". Please remove the storageClassName and size properties from your spec [...]`. The bug is fixed. However, if you receive this message, you need to perform the steps described in the link:https://access.redhat.com/solutions/6995414[Knowledgebase article]. (ROX-14335)
 * The `v1/alerts/summary/counts` endpoint, which is used by the *Policy violations by severity* and *Policy violations by category* widgets on the Dashboard, was fixed to prevent Central from crashing in highly-scaled environments. (ROX-13829)
 * {product-title-short} previously was not able to correctly identify the signature of images that contained the same content but were stored in different registries, causing problems with alerts and enforcement. This issue has been fixed and {product-title-short} now correctly identifies signatures of images from different registries. (ROX-12400)
 
 [id="resolved-in-version-3741"]
 === Resolved in version 3.74.1
 
+Release date: 20 March 2023
+
 * Previously, Sensor that was installed on secured clusters could not connect to Central when using a proxy, even if the proxy was specified in the proxy configuration file or in environment variables. This issue is fixed and Sensor on secured clusters now connects to Central when using a proxy. (ROX-15279)
 
+[id="resolved-in-version-3742"]
+=== Resolved in version 3.74.2
+
+Release date: 13 April 2023
+
+* This release of {product-title-short} includes a fix for link:https://access.redhat.com/errata/RHSA-2023:1405[RHSA-2023:1405] OpenSSL security update for {op-system-base-full} 8.
+* This release fixes a crash that occurs during migration to the PostgreSQL Technology Preview when there are clusters that have not checked in recently.
```

Comment:
I think this sentence might need revised:

This release fixes a crash that occurs during migration to the PostgreSQL Technology Preview when there are clusters that have not checked in recently.

Is the PostgreSQL DB in tech preview for RHACS? Should it be like 

* This release fixes a crash that occurs during migration to the PostgreSQL database when there are. . . 

? cc @connorgorman

Reply:
Yes, it's still in tech preview